src/test/java/edu/ucsb/cs156/frontiers/WebTestCase.java

```diff
@@ -24,7 +25,8 @@
 import static com.github.tomakehurst.wiremock.core.WireMockConfiguration.options;
 
 @ActiveProfiles("integration")
-@DirtiesContext(classMode = DirtiesContext.ClassMode.BEFORE_EACH_TEST_METHOD)
+@DirtiesContext(classMode = DirtiesContext.ClassMode.BEFORE_CLASS)
+@ResourceLock("port-8080")
```

Comment:
Are these changes needed for this PR?  

Or is this a separate issue that should be in it's own PR with a separate explanation?